projects/flimnap/main.cc

```diff
@@ -66,8 +66,8 @@ namespace {
     constexpr float fixed_velocity = 5.0f;
 
     // transmission
-    std::array<uint8_t, SimulationMessage_size + packet::frame::PACKET_OVERHEAD> encode_buffer;
-    std::array<uint8_t, SimulationMessage_size + packet::frame::PACKET_OVERHEAD> packet_buffer;
+    std::array<uint8_t, SimulationMessage_size> encode_buffer;
```

Comment:
The varint can be up to 10 bytes according to the [nanopb documentation](https://jpa.kapsi.fi/nanopb/docs/reference.html#pb-encode-varint).